app/src/main/java/com/example/playlistmaker/media/ui/fragments/CreatingPlaylistFragment.kt

```diff
@@ -121,52 +114,50 @@ class CreatingPlaylistFragment : Fragment() {
                 //обрабатываем событие выбора пользователем фотографии
                 uriString = uri.toString()
                 if (uri != null) {
-                    binding.albumCoverage.setImageURI(uri)
                     binding.albumCoverageAdd.visibility = View.GONE
                     playlist.artworkUrl100 = uri.toString()
-                    viewModel.saveImageToPrivateStorage(uri, requireActivity())
+                    viewModel.saveImageToPrivateStorage(uri)
+                    Glide.with(requireActivity())
```

Comment:
🍏 Вместо разделения загрузки изображения с помощью if , используй метод placeholder, который подставит дефолтную картинку, если загрузка не удалась:
```suggestion
                    Glide.with(binding.albumCoverage)
                        .load(uri)
                        .placeholder(R.drawable.creating_album_cover)
                        .centerCrop()
                        .transform(
                            CenterCrop(),
                            RoundedCorners(resources.getDimensionPixelSize(R.dimen.player_album_cover_corner_radius))
                        )
                        .into(binding.albumCoverage)
```